tests/array_interoperability_test.py

```diff
@@ -174,12 +174,23 @@ def testTensorFlowToJaxInt64(self):
   @jtu.sample_product(
     shape=all_shapes,
     dtype=numpy_dtypes,
+    gpu=[False, True] if jtu.test_device_matches(["gpu"]) else [False],
```

Comment:
Turns out the advice I gave here is problematic: we can't call `jtu.test_device_matches` at the top level, because under `absl` the information is not yet available when the test file is imported.

I think I'd probably just remove the `gpu` parameter, and instead just test on the default defice. We'll get CPU coverage in CPU runtimes, and GPU coverage in GPU runtimes.

Reply:
Done!